From 2e46dd062470b07d1c297275ab5a7a2040e1ab80 Mon Sep 17 00:00:00 2001
From: Luca Weiss <luca@z3ntu.xyz>
Date: Thu, 30 Nov 2023 21:36:56 +0100
Subject: [PATCH] freedreno: Enable A305B

Enable the Adreno 305B that is found on the MSM8226(v2) SoC (Snadragon
400).

Signed-off-by: Luca Weiss <luca@z3ntu.xyz>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/26434>
---
 src/freedreno/common/freedreno_devices.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/freedreno/common/freedreno_devices.py b/src/freedreno/common/freedreno_devices.py
index 8661e90196a..6fe9caae83a 100644
--- a/src/freedreno/common/freedreno_devices.py
+++ b/src/freedreno/common/freedreno_devices.py
@@ -202,6 +202,7 @@ add_gpus([
         GPUId(307),
         GPUId(320),
         GPUId(330),
+        GPUId(chip_id=0x03000512, name="FD305B"),
     ], GPUInfo(
         CHIP.A3XX,
         gmem_align_w = 32,  gmem_align_h = 32,
-- 
2.43.0

